Source/WebCore/rendering/StyledMarkedText.cpp

```diff
@@ -136,11 +143,42 @@ Vector<StyledMarkedText> StyledMarkedText::subdivideAndResolve(const Vector<Mark
         return { styledMarkedText };
     }
 
-    auto markedTexts = MarkedText::subdivide(textsToSubdivide);
+    auto markedTexts = MarkedText::subdivide(textsToSubdivide, OverlapStrategy::None);
     ASSERT(!markedTexts.isEmpty());
     if (UNLIKELY(markedTexts.isEmpty()))
         return { };
 
+    if (!markedTexts.isEmpty()) {
+        // Get rid of duplicates in vector
+        ListHashSet<MarkedText> noDuplicates;
+        for (auto& elem : markedTexts)
```

Comment:
```suggestion
        for (auto& element : markedTexts)
```

Here and throughout the file, please don't use abbreviations.

Reply:
Got it!